modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,24 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords:
+
+== Premium upgrade promotion defaults
+
+When {productname} 6.2 or later is running in self-hosted Community mode, an *Upgrade* promotion button appears in the top-right of the editor menu bar by default. The button does note appear when {productname} is running in Community mode on the Tiny cloud.
```

Comment:
```suggestion
As part of the 6.2 release, self-hosted {productname} now features a promotional button labelled *Upgrade* which redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/?utm_source=TinyMCE&utm_medium=SPAP&utm_campaign=SPAP&utm_id=editorreferral[promotion page], informing self-hosted users of the benefits of switching to {productname} cloud. This button is only visible in the self-hosted {communityname} version of the editor, and will not appear in any {cloudname} or self-hosted Premium versions of the editor.

The button can be removed by setting the `+promotion+` option to `+false+`.
```

Reply:
To quote my commit message:

> Added extra explanatory material regarding what the promotion button does.
> 
> Core text provided by @MitchCTiny. The edit and re-arrangement is my fault.

Reply:
Should this really be linking with the exact same URL? That seems like a very bad idea will mess with Marketing analytics.

Reply:
I just copied-and-pasted, but I’m entirely happy to reduce that to the bare URL (ie, delete everything from the `?` to the end of the string).

Reply:
Ah right sorry I just grabbed the link since I figured it would be good to acknowledge it on this page.

Reply:
And done: the URL is now bare.

Reply:
Marketing may want a different set of query parameters for the docs, but I guess they'll have the docs in the referrer header which might be enough.